website/README.md

```diff
@@ -0,0 +1,41 @@
+# Website
+
+This website is built using [Docusaurus 2](https://docusaurus.io/), a modern static website generator.
+
+### Installation
+
+```
+$ yarn
+```
+
+### Local Development
+
+```
+$ yarn start
+```
+
+This command starts a local development server and opens up a browser window. Most changes are reflected live without having to restart the server.
+
+### Build
+
+```
+$ yarn build
+```
+
+This command generates static content into the `build` directory and can be served using any static contents hosting service.
+
+### Deployment
```

Comment:
It would be great to have an automation that autogenerates the docs when we push to `main`. I guess we don't have this in js-composedb yet either @PaulLeCam?

Reply:
Yes just added!
https://github.com/ceramicstudio/js-composedb/blob/main/.github/workflows/deploy-docs.yml to deploy when pushing to `main` and https://github.com/ceramicstudio/js-composedb/blob/main/.github/workflows/build-docs.yml to check the website correctly builds when opening PRs to `main`

Reply:
Thanks! WIll add it here too!